hr_timesheet_improvement/tests/test_timesheet_improvement.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+#
+#
+#    Authors: Damien Crier
+#    Copyright 2015 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+
+import datetime
+import time
+from openerp import fields, exceptions
+from openerp.tests import common
+from openerp.tools import DEFAULT_SERVER_DATETIME_FORMAT
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT
```

Comment:
You should remove these two last import and use functions `from_string` and `to_string` defined on on classes `fields.Date` and `fields.Datetime` to convert your datetime objects in the from/to the format expected by the server.